pkgs/os-specific/linux/bolt/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, meson, ninja, pkgconfig, fetchFromGitLab,
+  python3, umockdev, gobject-introspection,
+  git, asciidoc, libxml2, libxslt, docbook_xml_dtd_45, docbook_xsl,
+  glib, systemd, polkit
+}:
+
+stdenv.mkDerivation rec {
+  name = "bolt-${version}";
```

Comment:
It is now possible to use

```suggestion
  pname = "bolt";
```

Like in python packages.

Reply:
Oh, great! Will do.